src/niquests/adapters.py

```diff
@@ -1682,6 +1685,9 @@ def cert_verify(
                     if isinstance(verify, str):
                         cert_loc = verify
 
+                    elif isinstance(verify, Path):
```

Comment:
It would be better to check for attr `__fspath__` existence when updating to PathLike.

Reply:
Would immediately remove the nested calls `.absolute().as_posix()`.